src/components/BPJsonGenerator/BPJsonGenerator.js

```diff
@@ -429,19 +429,23 @@ const BPJsonForm = ({ accountName, bpJson, onSubmit }) => {
                 variant="outlined"
                 type="number"
                 label="Latitude"
-                value={org.location.latitude || ''}
+                value={org.location.latitude}
                 className={classes.formField}
               />
             </Grid>
             <Grid item xs={12} sm={6}>
               <TextField
                 onChange={(e) =>
-                  handleOnChange('longitude', e.target.value, 'location')
+                  handleOnChange(
+                    'longitude',
+                    Number(e.target.value),
```

Comment:
@JustinCast you don't need to parse to Number the latitude field? it looks like this right now

```
onChange={(e) =>
  handleOnChange('latitude', e.target.value, 'location')
}
```